apihub/subscription/schemas.py

```diff
@@ -17,14 +31,24 @@ class SubscriptionBase(BaseModel):
     tier: str
 
 
+class SubscriptionIn(BaseModel):
+    username: str
+    application: str
+    tier: str
+    credit: Optional[int] = 0
```

Comment:
possibly remove this one since it is not set by user?

Reply:
yes, we are setting it.
ok